src/debugAdapter/goDebug.ts

```diff
@@ -250,25 +269,6 @@ class Delve {
 				return;
 			}
 
-			// read env from disk and merge into envVars
-			if (launchArgs.envFile) {
-				try {
-					const buffer = stripBOM(FS.readFileSync(launchArgs.envFile, 'utf8'));
-					buffer.split('\n').forEach( line => {
-						const r = line.match(/^\s*([\w\.\-]+)\s*=\s*(.*)?\s*$/);
-						if (r !== null) {
-							let value = r[2] || '';
-							if (value.length > 0 && value.charAt(0) === '"' && value.charAt(value.length - 1) === '"') {
-								value = value.replace(/\\n/gm, '\n');
-							}
-							env[r[1]] = value.replace(/(^['"]|['"]$)/g, '');
```

Comment:
By the way, I'm suggesting that we use the following precedence order 
`launch env -> file env -> process env` so we can override file env values with launch specific values.

What do you think? Check the last commit.